.gitignore

```diff
@@ -1,14 +1,3 @@
-# message and service generated files
-msg_gen/
-msg_cs_gen/
-srv_gen/
-srv_cs_gen/
-proxy_gen/
-idl_gen/
```

Comment:
what's the motive behind removing these lines?

Reply:
I think they are not needed. srv_cs_gen for example is about the files that generated were generated when we used ROS + C# back in I think ~2010.